### PR TITLE
allow methods to use `de_train_h5ad` instead of `de_train`.

### PR DESCRIPTION
**Pull request by rcannood**
## Checklist before requesting a review
- [ ] I have performed a self-review of my code

- Check the correct box. Does this PR contain:
  - [ ] Breaking changes
  - [ ] New functionality (new method, new metric, ...)
  - [ ] Major changes
  - [ ] Minor changes
  - [ ] Bug fixes

- [ ] Proposed changes are described in the CHANGELOG.md

- [ ] CI Tests succeed and look good!

### CODE REVIEW

**Review by szalata — CHANGES_REQUESTED** [review]
We get an error in
def set_par_values(config) -> None:

due to the resources path based on par argument that isn't necessary.
```
❯ ./scripts/add_a_method.sh
[warning] --task looks like a parameter but is not a defined parameter and will instead be treated as a positional argument. Use --help to get more information on the parameters.
Check inputs
Check language
Check API file
Read API file
Create output dir
Create config
Create script
Traceback (most recent call last):
  File "/viash_automount/var/folders/98/9ygjwkpd12d1vxydfprn1st00000gp/T//viash-run-create_component-CTkCHC.py", line 358, in <module>
    main(par)
  File "/viash_automount/var/folders/98/9ygjwkpd12d1vxydfprn1st00000gp/T//viash-run-create_component-CTkCHC.py", line 342, in main
    set_par_values(api)
  File "/viash_automount/var/folders/98/9ygjwkpd12d1vxydfprn1st00000gp/T//viash-run-create_component-CTkCHC.py", line 194, in set_par_values
    value = f'resources_test/{par["task"]}/pancreas/{key_strip}.h5ad'
KeyError: 'task'
Files and logs are stored at '/tmp/viash_create_component12384494734812512647'
```

**Comment by szalata** [comment]
for the issue in `def set_par_values(config) -> None:`

I think we could replace with:

```
    # find value
    if arg["type"] != "file":
      value = arg.get("default", arg.get("example", "..."))
    elif key == "de_train":
      value = "resources/neurips-2023-kaggle/de_train.parquet"
    elif key == "de_train":
      value = "resources/neurips-2023-kaggle/de_train.h5ad"
    elif key == "id_map":
      value = "resources/neurips-2023-kaggle/id_map.csv"
    else:
      key_strip = key.replace("output_", "")
      value = f'{key_strip}.h5ad'
```

File: src/common/create_component/script.py
Comment (szalata) [inline src/common/create_component/script.py:1]: @rcannood could you please confirm whether that removal is ok?

**Review by szalata — APPROVED** [review]

**Comment by szalata** [comment]
after the 2 small changes I commented on above, lgtm!

File: src/common/create_component/script.py [hunk 1]
```diff
@@ -170,11 +170,10 @@ def set_par_values(config) -> None:
       value = arg.get("default", arg.get("example", "..."))
     elif key == "de_train":
       value = "resources/neurips-2023-kaggle/de_train.parquet"
+    elif key == "de_train":
```
Comment (rcannood) [inline src/common/create_component/script.py:173]: Good point! However, do you mean de_train_h5ad?

Comment (szalata) [reply src/common/create_component/script.py]: thanks for pointing out, just fixed

**Pull request merged**